src/main/java/com/aws/greengrass/secretmanager/SecretManagerService.java

```diff
@@ -101,6 +101,8 @@ protected void install() throws InterruptedException {
         setIsInitialSyncComplete(new CountDownLatch(1));
         // subscribe will invoke serviceChanged right away to sync from cloud
         this.config.lookupTopics(CONFIGURATION_CONFIG_KEY).subscribe(this.handleConfigurationChangeLambda);
+        // Re-initialize the cloud thread pool for ipc async request processing
+        secretManagerIPCAgent.setCloudCallThreadPool(config);
```

Comment:
should probably also requestReinstall if cloudRequestQueueSize changes

Reply:
Updated